src/mixins/raster-layer-point-mixin.js

```diff
@@ -581,7 +581,7 @@ export default function rasterLayerPointMixin(_layer) {
 
     const marks = [
       {
-        type: markType === "airplane" ? "legacysymbol" : "symbol",
+        type: "symbol",
```

Comment:
airplane symbol can use the normal `symbol` type now. that was added with this PR: https://github.com/omnisci/omniscidb-internal/pull/5660